app/page.tsx

```
@@ -1,101 +1,185 @@
+
+      {/* Footer */}
+      <footer className="relative z-10 border-t border-slate-800 py-12 px-6">
+        <div className="max-w-7xl mx-auto text-center">
+          <div className="text-3xl font-bold bg-gradient-to-r from-cyan-400 to-teal-400 bg-clip-text text-transparent mb-6">
+            Tradoxus
+          </div>
+          <p className="text-slate-400 mb-8">
+            Master crypto trading safely through gamified education
+          </p>
+          <div className="flex justify-center space-x-8 text-slate-400">
+            <a href="#" className="hover:text-cyan-400 transition-colors">
```

Biome (1.9.4)

[error] Provide a valid value for the attribute href.

The href attribute should be a valid a URL
Check this thorough explanation to better understand the context.

(lint/a11y/useValidAnchor)